pkg/plugin/aggregation/run.go

```diff
@@ -190,9 +192,17 @@ func Run(client kubernetes.Interface, plugins []plugin.Interface, cfg plugin.Agg
 		return errors.Wrapf(err, "couldn't get aggregator pod")
 	}
 
-	for _, p := range plugins {
-		logrus.WithField("plugin", p.GetName()).Info("Running plugin")
-		go aggr.RunAndMonitorPlugin(context.Background(), time.Duration(cfg.TimeoutSeconds)*time.Second, p, client, nodes.Items, cfg.AdvertiseAddress, certs[p.GetName()], aggregatorPod, progressPort, pluginResultsDir)
+	for _, pluginGroup := range getOrderedPlugins(plugins) {
+		var waitGroup sync.WaitGroup
+		waitGroup.Add(len(pluginGroup))
```

Comment:
I think it would be really beneficial as this feature gets adoption that we add some extra logging here. We need to somehow provide some info about the grouping before it happens. That way we can see clearly in the logs if there is an issue.

Some ideas:
 - `Two plugins with order %v, %v still waiting to be launched.`
 - `Launching plugin %v with order %v` (just adding the order info to the existing log line)
 - Before the loop at all we could just log the entire ordered plugins for maximum clarity: `Got plugin ordering: %v`